pkgs/by-name/im/import-linter/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  python3Packages,
+  fetchPypi,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "import-linter";
+  version = "2.5.2";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit version;
+    pname = "import_linter";
+    sha256 = "sha256-2PLcZDKXXMNe3EzAv88bgR8FUAs3fODD9icp1o9Gxpg=";
```

```suggestion
    hash = "sha256-2PLcZDKXXMNe3EzAv88bgR8FUAs3fODD9icp1o9Gxpg=";
```
the SRI format is already correct, no need to regenerate